src/config/keys.ts

```diff
@@ -1,13 +1,21 @@
 import { UUID } from '@graasp/sdk';
 
-import { SearchFields } from '../types';
+import { OwnItemsQuery, SearchFields } from '../types';
 import { AggregateActionsArgs } from '../utils/action';
 import { hashItemsIds } from '../utils/item';
 import { DEFAULT_THUMBNAIL_SIZE } from './constants';
 
 export const APPS_KEY = 'apps';
 export const ITEMS_KEY = 'items';
 export const OWN_ITEMS_KEY = [ITEMS_KEY, 'own'];
+export const buildOwnItemsKey = ({ page, name, all, limit }: OwnItemsQuery) => [
+  ITEMS_KEY,
+  'own',
+  page,
+  name,
+  all,
+  limit,
```

Comment:
It might make sense to have an object, so the parameters also makes more sense when we see a built key.
Also let's have two groupes as suggested in my previous comments.